README.md

```diff
@@ -53,7 +58,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+  var weather introduces the variable weather. weather = "sunny" makes weather equal to the string "sunny".  weather === "sunny" tell the expression to check if weather is equal to "sunny".
```

Comment:
Essentially...
It actually performs the comparison, though - that expression will evaluate to a boolean value, either `true` or `false`.